manifests/install.pp

```diff
@@ -18,7 +18,7 @@
   $python = $::python::version ? {
     'system' => 'python',
     'pypy'   => 'pypy',
-    default  => "${python::version}",
+    default  => "${python::version}", # lint:ignore:only_variable_string
```

Comment:
This should be converted to just be default => $python::version rather than disabling the line message.  Placing the variable in quotes like this does not cast it to a string, and the behavior both ways is identical.  You would need to use sprintf to cast it to a string, but that would actually change the current behavior.